File: ageguess.py
import random
import input
num = random.randrange(100)
guesses = 1
print(f"guess my age!")
while((guess:=input.num("your guess: ")) != num):
    if guess > num:
        print("too high!")
    else:
        print("too low!")
    guesses += 1
print(f"you got it! it took {guesses} tries.")

File: lua/AgeGuess.lua
local age = math.random(100)
print(age)
local guesses = 0

print("hi welcome to age guess game!")
print("guess my age")
while true do
    guesses = guesses + 1
    local guess = tonumber(io.read())
    if guess < age then
        print("too low! guess again")
    elseif guess > age then
        print("too high! guess again")
    else
        break
    end
    print("")
end
print("congrats, u guessed my age! it took " .. guesses .. " guesses!")